crates/sequencer/src/sequencer.rs

```diff
@@ -25,3 +36,173 @@ where
         Self { context, state }
     }
 }
+
+impl<S> Execution for Sequencer<S>
+where
+    for<'a> &'a mut S: State + StateReader + Committer<S>,
+{
+    fn execute(&mut self, transaction: Transaction) -> Result<(), TransactionExecutionError> {
+        let mut cached_state = CachedState::new(&mut self.state);
+        let res = transaction.execute(&mut cached_state, &self.context, false);
```

Comment:
can we name the boolean variable? so that reader can see at a glance what is this third argument?